nodes/datasets.py

```diff
@@ -199,8 +202,22 @@ def load(self, context: Context) -> ppl.PathsDataFrame:
             df = df.with_columns(pl.col(self.column).alias(VALUE_COLUMN))
             cols = [YEAR_COLUMN, VALUE_COLUMN, *df.dim_ids]
 
-        if YEAR_COLUMN in cols and YEAR_COLUMN not in df.primary_keys:
-            df = df.add_to_index(YEAR_COLUMN)
+        if YEAR_COLUMN in cols:
+            if YEAR_COLUMN not in df.primary_keys:
+                df = df.add_to_index(YEAR_COLUMN)
+            if len(df.filter(pl.col(YEAR_COLUMN).lt(100))) > 0:
+                baseline_year = context.get_parameter_value('baseline_year', required=True)
+                df = df.with_columns(
+                    pl.when(pl.col(YEAR_COLUMN).lt(100))
```

Comment:
I only have limited Polars knowledge, but it seems that this is equivalent and maybe easier to read: `pl.when(pl.col(YEAR_COLUMN) < 100)`